coursier.bzl

```diff
@@ -77,19 +77,21 @@ def _relativize_and_symlink_file(repository_ctx, absolute_path):
 # location of `java`.
 def _generate_java_jar_command(repository_ctx, jar_path):
     java_home = repository_ctx.os.environ.get("JAVA_HOME")
+    coursier_opts = repository_ctx.os.environ.get("COURSIER_OPTS")
```

Comment:
use the `default` param: https://docs.bazel.build/versions/2.1.0/skylark/lib/dict.html#get

Reply:
Changed default value to empty string. I still prefer returning in the next line an empty list if this option wasn't set, so I applied len(...) > 0.